apps/video/test/marionette/lib/video.js

```diff
@@ -130,14 +133,30 @@ Video.prototype = {
    * @return {Marionette.Element} First element of all thumbnail images.
    */
   get thumbnail() {
-    return this.client.findElement(Video.Selector.thumbnail);
+    return this.client.helper.waitForElement(Video.Selector.thumbnail);
   },
 
   /**
    * @return {Marionette.Element} video player element.
    */
   get player() {
-    return this.client.findElement(Video.Selector.player);
+    return this.client.helper.waitForElement(Video.Selector.player);
+  },
+
+  showControls: function() {
+    this.player.tap();
+  },
+
+  elapsedTime: function() {
+    var timeElem =
+      this.client.helper.waitForElement(Video.Selector.elapsedTime);
```

Comment:
If it is not displayed, this will block forever, I think. Maybe better to use findElement here?

I suppose that by doing this, it means that you don't need to have a wait after you call tap(). It would be cleaner to put the tap() in the ensureControlsShowing() function, though.